diff --git a/node_modules/@polkadot/apps-config/endpoints/index.js b/node_modules/@polkadot/apps-config/endpoints/index.js
index 30022a8..948d438 100644
--- a/node_modules/@polkadot/apps-config/endpoints/index.js
+++ b/node_modules/@polkadot/apps-config/endpoints/index.js
@@ -1,6 +1,6 @@
 // Copyright 2017-2021 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
-import { createCustom, createDev, createOwn } from "./development.js";
+import { createCustom, createDev } from "./development.js";
 import { createProduction } from "./production.js";
 import { createKusamaRelay, createPolkadotRelay } from "./productionRelays.js";
 import { createTesting } from "./testing.js";
@@ -68,7 +68,7 @@ export function createWsEndpoints(t, firstOnly = false, withSort = true) {
     }),
     textBy: '',
     value: ''
-  }, ...createDev(t), ...createOwn(t)].filter(({
+  }, ...createDev(t)].filter(({
     isDisabled
   }) => !isDisabled);
 }
\ No newline at end of file
