diff --git a/node_modules/react-native-haptic-feedback/android/build.gradle b/node_modules/react-native-haptic-feedback/android/build.gradle
index c326a26..dd33c61 100644
--- a/node_modules/react-native-haptic-feedback/android/build.gradle
+++ b/node_modules/react-native-haptic-feedback/android/build.gradle
@@ -1,6 +1,7 @@
 def DEFAULT_COMPILE_SDK_VERSION             = 27
 def DEFAULT_BUILD_TOOLS_VERSION             = "27.0.0"
 def DEFAULT_TARGET_SDK_VERSION              = 27
+def DEFAULT_MIN_SDK_VERSION                 = 16
 
 buildscript {
     repositories {
@@ -20,7 +21,7 @@ android {
     buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion rootProject.hasProperty('minSdkVersion') ? rootProject.minSdkVersion : DEFAULT_MIN_SDK_VERSION
         targetSdkVersion rootProject.hasProperty('targetSdkVersion') ? rootProject.targetSdkVersion : DEFAULT_TARGET_SDK_VERSION
         versionCode 1
         versionName "1.0"
@@ -38,4 +39,3 @@ repositories {
 dependencies {
     implementation 'com.facebook.react:react-native:+'
 }
-  
