diff --git a/node_modules/react-native-aes-crypto/index.d.ts b/node_modules/react-native-aes-crypto/index.d.ts
index 0e07c19..6edb226 100644
--- a/node_modules/react-native-aes-crypto/index.d.ts
+++ b/node_modules/react-native-aes-crypto/index.d.ts
@@ -1,4 +1,4 @@
-declare module '@tectiv3/react-native-aes-crypto' {
+declare module 'react-native-aes-crypto' {
     function pbkdf2(password: string, salt: string, cost: number, length: number): Promise<string>;
     function encrypt(text: string, key: string, iv: string): Promise<string>;
     function decrypt(ciphertext: string, key: string, iv: string): Promise<string>;
diff --git a/node_modules/react-native-aes-crypto/index.js b/node_modules/react-native-aes-crypto/index.js
index 037759c..3c7ff21 100755
--- a/node_modules/react-native-aes-crypto/index.js
+++ b/node_modules/react-native-aes-crypto/index.js
@@ -1,2 +1,4 @@
 'use strict';
 import { NativeModules } from 'react-native';
+
+export default NativeModules.Aes;
